ruby/ext/google/protobuf_c/extconf.rb

```diff
@@ -25,4 +25,4 @@
          "repeated_field.c", "map.c", "ruby-upb.c", "wrap_memcpy.c",
          "naive.c", "range2-neon.c", "range2-sse.c"]
 
-create_makefile(ext_name, Dir.pwd+"/../../../../ext/google/protobuf_c")
```

Comment:
I think this may have been necessary to get this file working correctly in a Bazel sandbox (looking at your failures).  Did your Bazel command fail from this or was it just manual runs?